sdk/search/azure-search-nspkg/setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import sys
+from setuptools import setup
+
+# azure v0.x is not compatible with this package
+# azure v0.x used to have a __version__ attribute (newer versions don't)
+try:
+    import azure
+
+    try:
+        ver = azure.__version__
+        raise Exception(
+            "This package is incompatible with azure=={}. ".format(ver) + 'Uninstall it with "pip uninstall azure".'
+        )
+    except AttributeError:
+        pass
+except ImportError:
+    pass
+
```

Comment:
https://github.com/Azure/azure-sdk-for-python/blob/c6be5cdf8074ab5e8a79893c12d56f922514d181/sdk/core/azure-nspkg/setup.py#L26-L31